src/include/simeng/CoreInstance.hh

```diff
@@ -75,6 +75,8 @@ class CoreInstance {
   /** Getter for the set simulation mode in a string format. */
   const std::string getSimulationModeString() const;
 
+  const std::string getISAString() const;
```

Comment:
Add quick comment: `/** Getter for the ISA in string format */`
Just makes this file more consistent, even if it's self explanatory